tianshou/data/batch.py

```diff
@@ -243,6 +243,10 @@ class Batch:
     Convenience helpers are available to convert in-place the stored data into
     Numpy arrays or Torch tensors.
 
+    For Numpy arrays, only data types with ``np.object`` and numbers are
+    supported. For strings or other data types, however, they can be held
+    in ``np.object`` arrays.
+
```

Comment:
Move to line 71 or 88

Reply:
Fixed in 0b5320e64aaf8ab36482cb50294fd4ed75bf0996 .